doc/manuals/sprokit/pipeline_design.rst

```diff
@@ -91,18 +92,19 @@ may feed into any number of edges.
 Ports
 -----
 
-Ports are declared within a process and managed by the base \ref
-sprokit::process class to minimize the amount of code that needs to be written
+Ports are declared within a process and managed by the base
+``sprokit::process`` class to minimize the amount of code that needs to be written
 to handle communication within the pipeline.
 
-A port has a "type" associated with it which is used to detect errors when
-connecting incompatible ports with each other. These types are \em logical
-types, not a type within a programming language. A \c double can represent a
-distance or a time interval (or even a distance is a different unit!), but a
-port which uses a \c double to a distance would have a type of \c
-distance_in_meters, \em not \c double. There are two special types, one of
-which indicates that any type is accepted on the port and another which
-indicates that no data is ever expected on the port.
+A port has a "type" associated with it which is used to detect errors
+when connecting incompatible ports with each other. These types are
+**logical** types, not a type within a programming language. A
```

Comment:
Why the change from emphasis (italic) to strong (bold)? (Especially as "not", later, is still emphasized?)